Foody/Onboarding/SecondScreen/SecondViewController.swift

```diff
@@ -8,11 +8,33 @@
 import UIKit
 
 class SecondViewController: UIViewController {
-
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var startButtonTapped: UIButton!
+    @IBOutlet weak var firstLabel: UILabel!
+    @IBOutlet weak var secondLabel: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
+        firstLabel.text = "No matter what I cook!"
+        firstLabel.font = UIFont(name: "Georgia",size: 25)
+        view.addSubview(firstLabel)
```

Comment:
Please remove this line. No need to add label to view programatically. When you use storyboard it is automatically happen.